src/cljdoc/server/routes.clj

```diff
@@ -41,8 +42,7 @@
     ["/versions/:group-id/:artifact-id" :get nop :route-name :artifact/index]})
 
 (defn open-search-routes []
-  #{["/search" :get nop :route-name :search]
-    ["/suggest" :get nop :route-name :suggest]})
+  #{["/search" :get nop :route-name :search]})
```

Comment:
Q: Rename `open-search-routes` to `search-pages-routes`?

Reply:
Sounds good 👍